src/commands/create.js

```diff
@@ -190,6 +191,19 @@ export default asyncCommand({
 			spinner.succeed('Done!\n');
 		}
 
+		if (argv.less || argv.sass || argv.stylus) {
+			let extension;
+
+			if (argv.less) extension = '.less';
+			if (argv.sass) extension = '.scss';
+			if (argv.stylus) extension = '.styl';
+
+			const cssFiles = await promisify(glob)(`${target}/src/**/*.css`);
```

Comment:
This will only work for the default template. Maybe we could glob the examples folder and replace result's base path with target? Dunno if it is worth the trouble. @developit we're gonna keep all the examples after 2.0?